STYLE.md

```diff
@@ -151,33 +196,39 @@ The following table contains some examples and how they render in the documentat
       <td><code>`Library.coroutine.create()`</code></td>
       <td><code><a href="https://create.roblox.com/docs/reference/engine/libraries/coroutine">coroutine.create()</a></code></td>
     </tr>
+   <tr>
+      <td><code>`Class.Beam:SetTextureOffset()|myBeam:SetTextureOffset(1)`</code></td>
+      <td><code><a href="https://create.roblox.com/docs/reference/engine/classes/Beam#SetTextureOffset">myBeam:SetTextureOffset(1)</a></code></td>
+    </tr>
+    <tr>
+      <td><code>`Datatype.CFrame.new(x, y, z)`</code></td>
+      <td><code><a href="https://create.roblox.com/docs/reference/engine/datatypes/CFrame#new">CFrame.new(x, y, z)</a></code></td>
+    </tr>
   </tbody>
 </table>
 
-## Referencing the API
+#### Link Substitution
 
-When referencing a term from the API in prose, use [automatic API links](#links) whenever it makes sense. The examples in this section demonstrate the API link syntax.
+If desired, you can override the default link text using a `|` character followed by the desired term:
 
-Not every mention of a class, method, or property needs an auto-link. Particularly in an ordered list, overuse of links can distract readers and pull them out of the task flow. In those situations, use your best judgment on whether a link is necessary and helpful. In other situations, such as the API documentation or a reference table, auto-links are almost always a good idea.
-
-If you refer to a term without linking to it, maintain the same capitalization and wrap it in backticks. For example, refer to a `BasePart`, not a "base part."
-
-Never frame API names in the possessive:
+- `` `Class.Name.Property|PropertyName` ``
+- `` `Enum.Name|EnumItemName` ``
 
-**Bad**: A `Class.BasePart`'s color is defined by its `Color` property.
+This technique should be used to shorten the length of rendered links on the page:
 
-**Better**: The color of a `Class.BasePart` is defined by its `Class.BasePart.Color` property.
+- **WORSE**: [`MarketplaceService`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService) is responsible for in-experience transactions. The most notable methods are [`MarketplaceService:PromptProductPurchase()`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService#PromptProductPurchase) and [`MarketplaceService:PromptPurchase()`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService#PromptPurchase), as well as the callback [`MarketplaceService.ProcessReceipt`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService#ProcessReceipt) which must be defined so that transactions do not fail.
```

Comment:
```suggestion
- **BAD**: [`MarketplaceService`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService) is responsible for in-experience transactions. The most notable methods are [`MarketplaceService:PromptProductPurchase()`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService#PromptProductPurchase) and [`MarketplaceService:PromptPurchase()`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService#PromptPurchase), as well as the callback [`MarketplaceService.ProcessReceipt`](https://create.roblox.com/docs/reference/engine/classes/MarketplaceService#ProcessReceipt) which must be defined so that transactions do not fail.
```